sunpy/timeseries/timeseriesbase.py

```diff
@@ -855,7 +855,7 @@ def to_array(self, **kwargs):
         Parameters
         ----------
         **kwargs : `dict`
-            All keyword arguments are passed to `pandas.DataFrame.to_numpy`.
+            All keyword arguments are passed to ``pandas.DataFrame.to_numpy``.
```

Comment:
This looks like a bug in the pandas docs? Go to https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html, find `to_numpy` and there's no link there. So i think worth reporting as a bug to them?

Reply:
Done at https://github.com/pandas-dev/pandas/issues/57016

Reply:
Thanks